infrastructure/ai/providers/vertex_ai_provider.py

```
@@ -0,0 +1,209 @@
+        model_id = self._get_resolved_model(config)
+        
+        gen_config = self._build_genai_config(config, prompt.system)
+        
+        # 使用 aio 异步客户端
+        response = await self.client.aio.models.generate_content(
+            model=model_id,
+            contents=prompt.user,
+            config=gen_config
+        )
+        
+        # 提取内容 (CodeRabbit: 处理安全拦截，防止 response.text 为空时崩溃)
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)